Fix typo (#262)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

When you take a 100 anonset coin, and you register it again for CoinJoin, then you get one coin with anonset 200, and one change with anonset 100.
This change has anonset 100 because it can be linked to the input of the second CoinJoin, but this coin has anonset 100 already.

> So, although this change can still reveal pre-mix history, because the pre-mix history is another CoinJoin, you cannot go further back.

Reviewer: ```suggestion
This change can still reveal premix history which is another CoinJoin, therefore you cannot go further back.
```
